adapters/nextmillennium/nextmillennium.go

```diff
@@ -161,16 +178,21 @@ func (adapter *adapter) MakeBids(internalRequest *openrtb2.BidRequest, externalR
 	}
 
 	bidResponse := adapters.NewBidderResponseWithBidsCapacity(1)
-
+	errors := []error{}
```

Comment:
Please use `var errors []error` to avoid memory allocations when there is no error.

Reply:
thanks for the note, I fixed it